doorknob/tests/test_foyer.py

```diff
@@ -1,8 +1,7 @@
 import json
 import unittest
 
-from doorknob.foyer import Scene
-
+from foyer import Scene
```

Comment:
Importing from a higher level package in Python 3 is pretty tricky, and when I ran ```pytest``` on my machine this line caused a ```ModuleNotFoundError```. However, could you try switching this line to ```from doorknob.foyer import Scene```. Then, please verify with ```pytest``` that all the errors are resolved.

Reply:
I had file not found error because they are not in the same package. That's why I moved `test_detect.py` to its parent folder.